scripts/bench/README.md

```diff
@@ -1,40 +1,30 @@
-Work-in-progress benchmarks.
+# React Benchmarking
 
-## Running the suite
+## Commands
 
-You'll need two folders to compare, each of them containing `react.min.js` and `react-dom-server.min.js`. You can run `npm run build` at the repo root to get a `build` folder with these files.
+```bash
+# will compare local repo vs remote merge base repo
+yarn bench
 
-For example, if you want to compare a stable verion against master, you can create folders called `build-stable` and `build-master` and use the benchmark scripts like this:
+# will comapre local repo vs remote merge base repo
+# this can significantly improve bench times due to no build
+yarn bench -- --skip-build
 
-```
-$ ./measure.py build-stable stable.txt build-master master.txt
-$ ./analyze.py stable.txt master.txt
-```
+# will only build and run local repo against benchmarks
```

Comment:
I don't fully understand what these two commands do. (`--local` and `--remote`)
If I run `--local` will it not benchmark remote at all? Or will it just skip the build for remote?

If it doesn't run remote, how would I interpret the benchmark? Does it mean I'll only get absolute numbers as there's nothing to compare to?